client/components/Cost.jsx

```diff
@@ -1,27 +1,25 @@
 import React from 'react';
```

Comment:
Please revert changes back to original. (I have similar fixes on my current branch.)